addon/components/au-input.js

```diff
@@ -12,6 +12,23 @@ export default class AuInput extends Component {
       !this.args.onChange ||
         (typeof this.args.onChange === 'function' && this.isMasked),
     );
+
+    deprecate(
+      '[AuInput] The `@value` argument is deprecated. Use the `value` attribute and the `{{on}}` modifier instead.',
+      this.args.value === undefined,
```

Comment:
This won't fail if the consumer provides `undefined` as a value, we should use `'value' in this.args` instead